cmd/frostfs-adm/internal/modules/morph/netmap_candidates.go

```diff
@@ -0,0 +1,31 @@
+package morph
+
+import (
+	netmapCore "github.com/TrueCloudLab/frostfs-node/pkg/core/netmap"
+	"github.com/TrueCloudLab/frostfs-node/pkg/morph/client/netmap"
+	utilCore "github.com/TrueCloudLab/frostfs-node/pkg/util"
+	"github.com/nspcc-dev/neo-go/pkg/rpcclient/invoker"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+func listNetmapCandidatesNodes(cmd *cobra.Command, _ []string) {
+	c, err := getN3Client(viper.GetViper())
+	utilCore.ExitOnErr(cmd, "can't create N3 client: %w", err)
+
+	inv := invoker.New(c, nil)
+
+	cs, err := c.GetContractStateByID(1)
+	utilCore.ExitOnErr(cmd, "can't get NNS contract info: %w", err)
+
+	nmHash, err := nnsResolveHash(inv, cs.Hash, netmapContract+".frostfs")
+	utilCore.ExitOnErr(cmd, "can't get netmap contract hash: %w", err)
+
+	res, err := inv.Call(nmHash, "netmapCandidates")
+	utilCore.ExitOnErr(cmd, "can't fetch list of network config keys from the netmap contract", err)
+	if res != nil {
```

Comment:
Is it possible to have `res == nil` and `err == nil` at the same time?

Reply:
Removed redundant check.